_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -0,0 +1,311 @@
+
+### Deploy Forseti in an Existing GKE Cluster
+
+Create a file named *main.tf* in an empty directory and add the following content.  Add the appropiate values for each of the input vairables (e.g. domain, gsuite_admin_email).
+
+```bash
+module "forseti-on-gke-existing-gke-cluster" {
+    source                           = "terraform-google-modules/forseti/google//examples/on_gke_existing_gke_cluster"
+    credentials_path                 = ""
+    forseti_client_service_account   = ""
+    forseti_client_vm_ip             = ""
+    forseti_cloudsql_connection_name = ""
+    forseti_server_service_account   = ""
+    forseti_server_storage_bucket    = ""
+    gke_cluster_name                 = ""
+    gke_cluster_location             = ""
+    gke_service_account              = ""
+    project_id                       = ""
+    suffix                           = ""
+}
+```
+
+Initialize the Terraform module.
+
+```bash
+terraform init
+```
+
+Apply the Terraform module.
+
+```bash
+terraform apply
+```
+
+## Deploying with Helm
+
+### Deploy Forseti in an Existing GKE Cluster (Helm)
+
+Forseti can be deployed on GKE without the use of Terraform if the following preconditions are met.
+
+1. Forseti has been deployed in a GCP project.
+2. A GKE cluster has already been deployed in GCP under the same organization as Forseti.
+3. The [GKE credentials](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) exist on the machine executing the deployement
+4. The [Helm binary](https://helm.sh/docs/using_helm/) is installed and initiallized on the machine executing the deployment.
+
+After these preconditions are met, add the *forseti-security-charts* Helm repo to your Helm environment.
+
+```bash
+helm repo add forseti-security https://forseti-security-charts.storage.googleapis.com/release
+```
+
+Create and download [IAM service account keys](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) in JSON format for the **forseti-server** and **forseti-client** service accounts.
+
+Follow the [chart installation instructions](https://github.com/forseti-security/helm-charts/tree/master/charts/forseti-security#installing-the-forseti-security-chart) to install Forseti on GKE.
+
+
+## Post Deployment Configuration Changes
+
+After deploying Forseti, the default server configuration defines
+
+```yaml
+    rules_path: /home/ubuntu/forseti-security/rules
+```
+
+This path doesn't exist in the container.  Instead, the value of `rules_path` must point to the rules path in the Forseti server GCS bucket.  The can be changed by executing the following.
+
+1. Obtain the *forseti_server_conf.yaml* file from the GCS bucket.
+```bash
+gsutil cp gs://[FORSETI_SERVER_BUCKET]/configs/forseti_server_conf.yaml forseti_server_conf.yaml
+```
+
+2. Edit the forseti_server_conf.yaml file so that the `rules_path` points to the rules folder in the GCS bucket.
+```yaml
+	rules_path: gs://[FORSETI_SERVER_BUCKET]/rules
```

Comment:
Maybe this is just Git, but there is a red box right before `rules_path:`. Do you see it?

Reply:
Yes.  I think I know what it is.  I'll fix it.